logbook-logstash-logback-encoder/src/main/java/org/zalando/logbook/lle/AutodetectPrettyPrintingMarker.java

```diff
@@ -0,0 +1,45 @@
+package org.zalando.logbook.lle;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.PrettyPrinter;
+
+import net.logstash.logback.marker.RawJsonAppendingMarker;
+
+/**
+ * 
+ * Auto-detecting pretty-printing Marker. Pretty-printing cannot be performed in advance.
+ *
+ */
+
+public class AutodetectPrettyPrintingMarker extends RawJsonAppendingMarker {
```

Comment:
Package private? Final?

Reply:
Can be final. Must not be private, as it is referred to in configuration. However this is more of a convenience class than anything.

Reply:
Let's make it `final` then and mark it with `@API(status=INTERNAL)`?